samples/bluetooth/central_nfc_pairing/CMakeLists.txt

```diff
@@ -10,7 +10,9 @@ cmake_minimum_required(VERSION 3.13.1)
 set(NRF_SUPPORTED_BOARDS
   nrf52dk_nrf52832
   nrf52840dk_nrf52840
+  nrf52833dk_nrf52833
   nrf5340pdk_nrf5340_cpuapp
+  nrf5340dk_nrf5340_cpuapp
```

Comment:
Is there an issue with _ST25R3911B_ when building for `nrf5340dk_nrf5340_cpuappns`?

Reply:
There isn't issue but the overlay files have to be prepared for `nrf5340dk_nrf5340_cpuappns`